src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,57 @@
+.nav {
+  display: flex;
+  height: 60px;
+
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__item {
+    display: flex;
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding: 23px 0;
+    font-style: normal;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: #000;
+    text-decoration: none;
+  }
+
+  &__link:hover {
+    color: $light-blue-color;
+  }
+
+  &__link:active {
+    color: $light-blue-color;
+  }
+
+}
+
+.is-active {
+  color: $light-blue-color;
```

Comment:
you could add a class nav__link--active instead of these two classes because &__link:active works only when you are in the process of clicking on it